### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#306] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181301](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/c9441e5d-a942-4f9a-9bd9-adee96dfa893)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/58496088-b882-494a-b95d-8e8d6dc22f21)
**Soluzione alternativa**
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/f626b03c-2e08-47d2-ac49-e965b5bbe45e)

**Comment by Elia-Belli** [comment]
Siano $o(g)=n,o(\phi(g))=n'$:
**Oss:** $\phi(g^n)=\phi(0_G)=0_{G'}$ (abbiamo dimostrato che un omomorfismo mappa l'elemento neutro di $G$ a quello di $G'$)
$(\phi(g))^{n'}=0_{G'}$ per definizione di ordine
$\phi(g^n)=\phi(g)\star ... \star \phi(g)=(\phi(g))^{n}=0_{G'}$ ma $\phi$ è un isomorfismo $\Rightarrow n=n'$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-11 163912](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/66b6b0f4-056b-47cd-8f8f-2936ea560c5f)
![Immagine 2023-12-11 163928](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/447652ec-a096-4318-b135-0fa6ea7474a7)
